resources/content/config-content/bootstrap/bootstrap.sh

```diff
@@ -55,6 +55,19 @@ cleanup()
     return $exit
 }
 
+ca_cert()
+{
+    mkdir -p /usr/share/ca-certificates/rancher
+    cat > /usr/share/ca-certificates/rancher/agent-ca.crt << EOF
+%CERT%
+EOF
+    if ! grep -q rancher/agent-ca.crt /etc/ca-certificates.conf; then
```

Comment:
hmph. didnt know you could do this with the surrounding [ ]. Is that because ! is a construct similar to [] in how it operates?